plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -24,6 +24,10 @@ def deploy
       resources.each(&:deploy)
     end
 
+    def deploy_group_role
+      @deploy_group_role ||= DeployGroupRole.where(kubernetes_role: kubernetes_role, deploy_group: deploy_group).first
```

Comment:
should always exist, so add `!` ?
```suggestion
      @deploy_group_role ||= DeployGroupRole.where(kubernetes_role: kubernetes_role, deploy_group: deploy_group).first!
```

Reply:
It's technically possible that if you load up an old ReleaseDoc, it could be for a Role or DeployGroup that has since been (soft) deleted.  But I don't know what the expected behavior should be in that case.

Reply:
we would never recreate templates for old docs ... so make it blow up :)

Reply:
`!` ?